### PR TITLE
Rename complaint tool as feedback tool

### PR DESCRIPTION
**Pull request by cehsu**
## Description
These changes update the vets-json-schema version, the formId and the folders and file names.

## Testing done
Tests updated

#### Applies to all PRs

- [ ] Appropriate logging
- [x] Documentation has been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Review by erikphansen — APPROVED** [review]

File: src/applications/edu-benefits/feedback-tool/config/form.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ const formConfig = {
   trackingPrefix: 'gi_bill_feedback',
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
-  formId: 'complaint-tool',
+  formId: 'COMPLAINT-TOOL', // TODO: rename to feedback-tool once renamed on BE
```
Comment (cehsu) [inline src/applications/edu-benefits/feedback-tool/config/form.js]: We will work with @lihanli and @jholton to coordinate updating this with the BE changes in another PR.

Comment (cehsu) [reply src/applications/edu-benefits/feedback-tool/config/form.js]: Making this uppercase resolves the prefill issue for now.
